consensus/e2e_tests/utils_test.go

```diff
@@ -375,37 +374,73 @@ func basePersistenceMock(t *testing.T, _ modules.EventsChannel, bus modules.Bus)
 
 	blockStoreMock := persistenceMocks.NewMockBlockStore(ctrl)
 
-	blockStoreMock.EXPECT().Get(gomock.Any()).DoAndReturn(func(height []byte) ([]byte, error) {
-		heightInt := utils.HeightFromBytes(height)
-		if bus.GetConsensusModule().CurrentHeight() < heightInt {
-			return nil, fmt.Errorf("requested height is higher than current height of the node's consensus module")
-		}
-		blockWithHeight := &coreTypes.Block{
-			BlockHeader: &coreTypes.BlockHeader{
-				Height: utils.HeightFromBytes(height),
-			},
-		}
-		return codec.GetCodec().Marshal(blockWithHeight)
-	}).AnyTimes()
+	blockStoreMock.
+		EXPECT().
+		StoreBlock(gomock.Any(), gomock.Any()).
+		DoAndReturn(func(height []byte, block *coreTypes.Block) error {
+			return nil
+		}).
```

Comment:
Optional NIT: if there's nothing you actually want to do, I think you might be able to get away with just `Return(nil)`. Not 100% sure thought so you'd need to double check.